File: app/src/main/java/org/apache/fineract/data/services/AnonymousService.java
package org.apache.fineract.data.services;

import org.apache.fineract.data.models.customer.Country;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;

/**
 * @author Rajan Maurya
 *         On 26/07/17.
 */
public interface AnonymousService {

    @GET("http://restcountries.eu/rest/v2/all?fields=name;alpha2Code;translations")
    Observable<List<Country>> getCountries();
}

File: app/src/main/java/org/apache/fineract/data/services/AnonymousService.kt
package org.apache.fineract.data.services

import io.reactivex.Observable
import org.apache.fineract.data.models.customer.Country
import retrofit2.http.GET

/**
 * Created by Ahmad Jawid Muhammadi on 2/6/20
 */
interface AnonymousService {

    @GET("http://restcountries.eu/rest/v2/all?fields=name;alpha2Code;translations")
    fun getCountries(): Observable<List<Country?>?>?
}